### PR TITLE
Migrating an Angular 1 App to Angular 2 - Part 1

### PR DESCRIPTION
**Pull request by kmaida**
Post series: part 1 of 3.

### CODE REVIEW

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md [hunk 1]
```diff
@@ -0,0 +1,855 @@
...
+
+---
+
+## Angular 1 and Angular 2
+
+[AngularJS 1.x](https://angularjs.org/) has been a frontrunner among JavaScript frameworks over the past few years. There are thousands of production sites and apps [built with Google's "superheroic MVW framework"](https://www.madewithangular.com) and many more still in development. In mid-September 2016, [Angular 2 was released](http://angularjs.blogspot.com/2016/09/angular2-final.html) after a lengthy period of betas and release candidates. Angular developers knew this was coming and that [Angular 2](https://angular.io/) was a full rewrite and platform implementation, not an incremental update. 
+
+While Angular developers were and _are_ eager to try Angular 2, adoption can be challenging. Many of us have Angular 1 apps in development or maintenance and aren't in a position to migrate them to Angular 2 due to tight deadlines, budget constraints, client or management reluctance, etc. Angular 1 is still being maintained and Angular 1 apps are not about to go away.
```
Comment (kukicado) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: You can also mention that Angular 1.6 is landing soon. https://github.com/angular/angular.js/blob/master/CHANGELOG.md

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md [hunk 2]
```diff
@@ -0,0 +1,855 @@
...
+
+> **NOTE:** [Angular 2 uses SemVer (Semantic Versioning)](http://angularjs.blogspot.com/2016/10/versioning-and-releasing-angular.html?view=classic). This means that unlike Angular 1, there will no longer be breaking changes in point releases.
```
Comment (kukicado) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: If you want you can also say that Angular 3.0 will be released early next year. :)

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md [hunk 3]
```diff
@@ -0,0 +1,855 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="author" content="Auth0">
+  <meta name="description" content="Learn about some popular as well as obscure dinosaurs!">
+  <base href="/">
+
+  <!-- External stylesheets -->
+  <link 
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+    integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
```
Comment (kukicado) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: I think you can remove this integrity and cross origin check. Will help avoid confusion for newer reader.

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md [hunk 4]
```diff
@@ -0,0 +1,855 @@
...
+> **NOTE:** When moving or adding new files, you'll need to stop and restart the Angular CLI server (`Ctrl+C`, `ng serve`) to avoid module build errors. Changes within files are watched and live reloaded, but reorganizing the file structure can break this.
+
+Now let's add some global SCSS from ng1-dinos. We'll copy the files and subdirectories from [`ng1-dinos/src/assets/css/scss/core/`](https://github.com/auth0-blog/ng1-dinos/tree/master/src/assets/css/scss/core) to `ng2-dinos/src/assets/scss/`.
+
+> **NOTE:** If you paid close attention, you'll notice that we're left off a folder in ng2-dinos. Our Angular 1 ng1-dinos app had a `css` folder with `scss` inside it. We don't need the `css` folder in ng2-dinos because of the Angular CLI Webpack bundling.
+
+When we're done, our ng2-dinos global styles file structure should look like this:
+
+```text
+ng2-dinos
```
Comment (kukicado) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: typically for code samples on the blog it's recommended to only have 2 spaces (vs 4). This makes it much easier to read the code and doesn't force the user to have to horizontally scroll.

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md (hunk 1 above)
Comment (kmaida) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: 👍

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md (hunk 2 above)
Comment (kmaida) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: I'll mention this if I can find a link to information about it, but Googling didn't show me anything-- do you know of somewhere the Angular team mentions this date for 3.0?

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md (hunk 3 above)
Comment (kmaida) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: I'm just going to add a comment about copying Bootstrap from the Bootstrap CDN so people know where it came from. This is a pretty advanced tutorial, so I don't really want to leave anything out that people would do in a real-world app.

File: _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md (hunk 4 above)
Comment (kmaida) [inline _posts/2016-11-07-migrating-an-angular1-app-to-angular2-part-1.md]: I was trying to make the file structure blocks look nice, but they are too wide, agree! Fixed, I think they're more condensed now but still look good. 👍

**Pull request merged**